db_stress_tool/db_stress_test_base.cc

```diff
@@ -2423,6 +2427,80 @@ Status StressTest::TestGetPropertiesOfAllTables() const {
   return db_->GetPropertiesOfAllTables(&props);
 }
 
+void StressTest::TestGetUpdatesSince(ThreadState* thread) {
+  Status s = db_->DisableFileDeletions();
+  if (!s.ok()) {
+    return;
+  }
+
+  const SequenceNumber since_seq =
+      thread->rand.Next64() % (db_->GetLatestSequenceNumber() + 1);
+  std::unique_ptr<TransactionLogIterator> iter;
+
+  s = db_->GetUpdatesSince(since_seq, &iter);
+
+  if (!s.ok()) {
+    fprintf(stderr,
+            "Failed to call GetUpdatesSince(): %s with specified "
+            "seq num: %" PRIu64 "\n",
+            s.ToString().c_str(), since_seq);
+    thread->shared->SafeTerminate();
+  }
+
+  if (!iter->status().ok()) {
+    fprintf(stderr,
+            "Iterator with non-ok status returned by "
+            "GetUpdatesSince(): %s \n",
+            iter->status().ToString().c_str());
+    thread->shared->SafeTerminate();
+  } else if (!iter->Valid()) {
+    return;
+  }
+
+  BatchResult res = iter->GetBatch();
+  SequenceNumber last_seqno = res.sequence;
+  iter->Next();
```

Comment:
This seems like a lot of extra code to avoid a special case for the first iteration of the loop. Can we initialize `last_seqno` to zero and let the loop treat that as a value that does not need consecutive-ness checks?